cli/cmd/component.go

```diff
@@ -1215,3 +1181,41 @@ func downloadProgress(complete chan int8, path string, sizeB int64) {
 		time.Sleep(time.Second)
 	}
 }
+
+func LoadCatalog() (*lwcomponent.Catalog, error) {
+	cli.StartProgress("Loading component Catalog...")
```

Comment:
nitpick - only capitalize when needed, Catalog is not a thing that the customer knows IMO
```suggestion
	cli.StartProgress("Loading component catalog...")
```